src/main/java/uk/gov/companieshouse/api/accounts/links/SmallFullLinkType.java

```diff
@@ -19,7 +19,8 @@
     FIXED_ASSETS_INVESTMENTS_NOTE("fixed_assets_investments_note"),
     EMPLOYEES_NOTE("employees_note"),
     CURRENT_PERIOD_PROFIT_LOSS("current_period_profit_and_loss"),
-    PREVIOUS_PERIOD_PROFIT_LOSS("previous_period_profit_and_loss");
+    PREVIOUS_PERIOD_PROFIT_LOSS("previous_period_profit_and_loss"),
+    PROFIT_AND_LOSS("profit_and_loss");
```

Comment:
This doesn't need to be here, reference either the current or previous period variant of profit and loss